examples/server-demo/src/App.js

```diff
@@ -0,0 +1,70 @@
+import algoliarecommend from '@algolia/recommend';
+import {
+  FrequentlyBoughtTogether,
+  RelatedProducts,
+  TrendingItems,
+} from '@algolia/recommend-react';
+import React from 'react';
+
+import GridView from './GridView';
+import ProductItem from './ProductItem';
+
+const appId = 'XX85YRZZMV';
+const apiKey = '098f71f9e2267178bdfc08cc986d2999';
+const indexName = 'test_FLAGSHIP_ECOM_recommend';
+const recommendClient = algoliarecommend(appId, apiKey);
+
+const App = ({ initialState }) => {
+  return (
+    <div style={{ margin: '0 auto', maxWidth: '1100px' }}>
+      <h2 style={{ marginBottom: '0.5rem', marginTop: '1rem' }}>
+        Algolia Recommend SSR
+      </h2>
+      <div style={{ marginTop: '2rem' }}>
+        <TrendingItems
+          recommendClient={recommendClient}
+          indexName={indexName}
+          itemComponent={ProductItem}
+          maxRecommendations={5}
+          view={GridView}
+          translations={{
+            title: `Trending products`,
+          }}
+          initialState={initialState}
+        />
+      </div>
+
+      <div style={{ marginTop: '4rem' }}>
+        <RelatedProducts
+          recommendClient={recommendClient}
+          indexName={indexName}
+          objectIDs={['M0E20000000DWIV']}
+          itemComponent={ProductItem}
+          maxRecommendations={5}
+          view={GridView}
+          translations={{
+            title: 'Related products',
+          }}
+          initialState={initialState}
+        />
+      </div>
+
+      <div style={{ marginTop: '2rem' }}>
+        <FrequentlyBoughtTogether
+          recommendClient={recommendClient}
+          indexName={indexName}
+          objectIDs={['M0E20000000DWIV']}
+          itemComponent={ProductItem}
+          maxRecommendations={5}
+          view={GridView}
+          translations={{
+            title: 'Frequently bought together',
+          }}
+          initialState={initialState}
```

Comment:
this is actually the initial results, not the initial state, right? Not sure if the name is chosen correctly

Reply:
yes, those would be the initialResults for now. they are injected as a prop, we want to use this prop moving fwd for SSR, injecting these initialResults from the server (so technically it would be an initialState of the app at first load? 🤔 )

Reply:
It would still be the initial results when it's using real server side rendering no? the state is which parameters get sent, and the results are what is displayed. Especially this code for me indicates it's only results: 

https://github.com/algolia/recommend/pull/109/files#diff-22a64e6cc4de6dd486bbe8a1b8b0a65d2ad9f2564e01becb38de38d3a5af11e2R38-R40